### DISCUSSION TITLE
Volume of HSR voice

### DISCUSSION THREAD

**Discussion by jjnak** [HumanSupportRobot/QandA#84] [Q&A] [answered]
I would like to know how to change the volume of the HSR voice.

I changed the setting of /etc/pulse/system.pa to change the volume of the HSR speaker as follows, but there was no effect on the volume of the HSR voice. I noticed that the volume of the initial sound when booting the robot was reduced by this change.

set-sink-volume alsa_output.usb-TOYOTA_MOTOR_CORPORATION_TMC_SHM-005_Alpha-00.mono-fallback 50000 (from 80000 to 50000).

Your help would be highly appreciated.

-Jun

**Comment by kosei1515** [comment]
Hello.
Could you please refer this documentation?
https://docs.hsr.io/hsr_develop_manual/reference/packages/tmc_talk_hoya_py/PKGDOC.html?highlight=volume

In `tmc_talk_hoya_py` node,  you can set the `volume` parameter. The range is 0-500(%).
You can change this parameter with `launch` file or `rosparam set` command  and so on. 
http://wiki.ros.org/rosparam
http://wiki.ros.org/roslaunch/XML/rosparam

Thank you.

**Reply by jjnak** [reply]
Thank you for your reply. I would like to change this parameter when booting the robot. 

How I can set this parameter in the 
/etc/opt/tmc/robot/params.yaml
file on HSR? 

Is it correct to describe as follows:
/tmc_talk_hoya_py/volume: 50
to reduce the volume to 50%?

**Reply by yoshimi-iyoda** [reply, marked as answer]
The parameter file is the file of that name.

Since the node name is `/text_to_speech`, the parameters are as follows.
`/text_to_speech/volume: 50`

I have not been able to confirm that it is exactly 50%, but this will reduce the volume.

**Reply by jjnak** [reply]
Dear Iyoda-san,

Thank you for your response. By following your instructions, I was able to reduce the volume of HSR's voice.

-Jun
